database/src/test/scala/za/co/absa/atum/database/runs/GetPartitioningCheckpoints.scala

```diff
@@ -0,0 +1,352 @@
+package za.co.absa.atum.database.runs
+
+import za.co.absa.balta.DBTestSuite
+import za.co.absa.balta.classes.JsonBString
+import za.co.absa.balta.classes.setter.CustomDBType
+
+import java.time.OffsetDateTime
+import java.util.UUID
+import scala.util.Random
+
+class GetPartitioningCheckpoints extends DBTestSuite{
```

Comment:
Only really serious one. Please rename the class and file to `GetPartitioningCheckpointsTest`